### DISCUSSION TITLE
posts/2021/04/using-eleventys-official-image-plugin/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#14] [Announcements]
# posts/2021/04/using-eleventys-official-image-plugin/

The what, why, and how of Eleventy Image.

https://www.brycewray.com/posts/2021/04/using-eleventys-official-image-plugin/

**Comment by RichardJECooke** [comment]
But people don't keep their images all in one /img folder, they keep them with their posts

**Reply by brycewray** [reply]
Depends on the user (Eleventy doesn’t require this); but, in that case, you could adjust the shortcode so that `src` doesn’t include a fixed path, letting you supply the full path when you supply `src`. If you will **always** be keeping images with their posts, you could set that in the original shortcode with the appropriate [page variable](https://www.11ty.dev/docs/data-eleventy-supplied/).
